osu.Game/Overlays/HoldToConfirmOverlay.cs

```diff
@@ -51,7 +51,7 @@ private void load()
 
         protected override void Dispose(bool isDisposing)
         {
-            audio.Tracks.RemoveAdjustment(AdjustableProperty.Volume, audioVolume);
+            audio?.Tracks.RemoveAdjustment(AdjustableProperty.Volume, audioVolume);
```

Comment:
`audio` is never null.
```suggestion
            audio.Tracks.RemoveAdjustment(AdjustableProperty.Volume, audioVolume);
```

Reply:
This is intentional for cases where there is an exception (and it never gets to BDL). Commonly experienced in tests. Basically we don't presume things always hit BDL for the sake of testing.

Reply:
Ah, I see.